data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Phenotype observations,,severity,ontology,,,CatalogueOntologies,Severity,,,,,http://purl.obolibrary.org/obo/HP_0012824,"A value that describes the intensity of the phenotype (e.g., ""Profound"")","Patient registry, FAIR Genomes"
+Disease history,,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C15607,History of diagnoses of an individual at a specific timepoints,"Patient registry, FAIR Genomes"
+Disease history,,part of clinical observation,ref,1,TRUE,,Clinical observations,,,,,http://purl.obolibrary.org/obo/NCIT_C87853,"A reference to a specific clinical observation (e.g., ""Ind001_Phen001"")","Patient registry, FAIR Genomes"
+Disease history,,disease,ontology,1,TRUE,CatalogueOntologies,Diseases,,,,,http://semanticscience.org/resource/SIO_001331,"The diagnosis assigned for this particular observation (e.g., ""Spinocerebellar ataxia type 34"")","Patient registry, FAIR Genomes"
+Disease history,,age group at onset,ontology,,,CatalogueOntologies,Age groups at onset,,,,,"http://www.ebi.ac.uk/efo/EFO_0004847,http://purl.obolibrary.org/obo/NCIT_C20587","A categorical value that groups this individual's age of onset into groups (e.g., ""Adult"", ""18-24 years"", etc.)","Patient registry, FAIR Genomes"
+Disease history,,age of onset,period,,,,,,,,,http://www.ebi.ac.uk/efo/EFO_0004847,"The age at which manifestations of the diseaese first appeared (e.g., ""P32Y6M1D"")","Patient registry, FAIR Genomes"
+Disease history,,age group at diagnosis,ontology,,,CatalogueOntologies,Age groups,,,,,"http://www.ebi.ac.uk/efo/EFO_0004918,http://purl.obolibrary.org/obo/NCIT_C20587","A categorical value that groups this individual's age at diagnosis into groups (e.g., ""Adult"", ""18-24 years"", etc.)","Patient registry, FAIR Genomes"
+Disease history,,age at diagnosis,period,,,,,,,,,http://www.ebi.ac.uk/efo/EFO_0004918,"The age at which the diagnosis was made (e.g., ""P32Y6M1D"")","Patient registry, FAIR Genomes"
+Disease history,,has family history,bool,,,,,,,,,http://purl.obolibrary.org/obo/HP_0032316,An indication if the diagnosis is present in one or more family members or other relatives,Patient registry
+Disease history,,disease severity,ontology,,,CatalogueOntologies,Severity,,,,,http://purl.obolibrary.org/obo/HP_0012824,"A value that describes the intensity of the disease (e.g., ""Profound"")","Patient registry, FAIR Genomes"
+Disease history,,disease stage,ontology,,,CatalogueOntologies,Disease stage,,,,,http://purl.obolibrary.org/obo/OBI_0000278,"A part of an occurrence of a disease process which is associated with position in the normal progression of the disease (e.g., ""acute onset"")","Patient registry, FAIR Genomes"
+Measurements,,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C25209,Additional clinical measurements collected as part of the clinical observation,"Patient registry, FAIR Genomes"
```

Comment:
If these are clinical measurements, part of clinical observation, maybe then rename to Clinical measurements?

Reply:
Or lift this up to relate to any Observation. N.B. again, we really should look at ISA-tab as they have a similar thing but call it 'Assay' instead of 'Observation'.

Reply:
Good point. Yes, I think it is supposed to be clinical measurements. We can change this in another PR.